server/controllers/finance/reports/cashReport/index.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * cashReport Controller
+ *
+ *
+ * This controller is responsible for processing cash report.
+ *
+ * @module finance/cashReport
+ *
+ * @requires lodash
+ * @requires lib/db
+ * @requires lib/ReportManager
+ * @requires lib/errors/BadRequest
+ */
+
+const _ = require('lodash');
+const db = require('../../../../lib/db');
+const ReportManager = require('../../../../lib/ReportManager');
+const BadRequest = require('../../../../lib/errors/BadRequest');
+
+const TEMPLATE_1 = './server/controllers/finance/reports/cashReport/report_combined.handlebars';
```

Comment:
Can these be named something more meaningful?  Like `TEMPLATE_COMBINED`?  The name `TEMPLATE_1` doesn't tell a developer a lot about what it does.